motif_inference/unite_motifs_of_biological_condition.py

```diff
@@ -87,12 +87,15 @@ def unite_clusters(motif_inference_output_path, meme_file, biological_condition,
             clusters_to_combine.append(cluster_names)
 
     logger.info(f'Sorting clusters by rank...')
+    half_num_samples = round(len(sample_names)/2)
```

Comment:
Should be floor as ceiling to avoid unpredicted behavior.
For example if we have 5 samples in BC, should threshold be 2 or 3 ?